src/components/indexpage-components/EducationCounters.tsx

```diff
@@ -9,22 +9,25 @@ const counterData = [
     title: 'home.counters.c1.title',
     isGatsbyLink: false,
     url: 'https://diplomaterv.vik.bme.hu/hu/Browse.aspx?d=MIT',
-    count: 498
+    count: 600,
+    suffix: '+'
     /* From diplomaterv portal, 2010-2021 Summer */
   },
   {
     title: 'home.counters.c2.title',
     isGatsbyLink: false,
     url: 'http://tdk.bme.hu/Browse/Papers?f=VIK&d=MIT',
```

Comment:
old link not valid anymore.
maybe just use this: https://tdk.bme.hu/browse
(in the new site search parameters are sent in post body)